js/GroupButton/Overview.md

```diff
@@ -1,18 +1,18 @@
 ---
 layout: post
-title: Overview
-description: overview
+title: Overview of JavaScript GroupButton Control | Syncfusion
+description: Learn here about overview support in Syncfusion Essential JavaScript GroupButton Control, its elements, and more.
```

Comment:
Learn here about overview of Syncfusion Essential JavaScript GroupButton Control, its elements, and more.

Reply:
Done